src/Entities/Entity.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace UKFast\SDK\Entities;
+
+abstract class Entity
+{
+    protected $guarded = [];
+
+    /**
+     * Datastore constructor.
+     * @param array $properties
+     */
+    public function __construct(array $properties = [])
+    {
+        $this->fill($properties);
+    }
+
+    /**
+     * Fill the object properties (except for guarded)
+     * @param array $properties
+     */
+    public function fill(array $properties)
+    {
+        foreach (array_diff_key($properties, array_flip($this->guarded)) as $property => $value) {
```

Comment:
Please could we have a unit test that makes sure an existing property is populated, a non-existent property is not populated and that the constructor calls this method?